src/wiseagents/cli/wise_agent_cli.py

```diff
@@ -47,12 +49,28 @@ def main():
     file_path = None
     default_file_path = "src/wiseagents/cli/test-multiple.yaml"
 
+    parser = argparse.ArgumentParser(prog="Wise Agent Argument Parser", description="Wise Agent CLI to run and manage Wise Agents", add_help="")
+    parser.add_argument("filename")
+    parser.add_argument("--debug", dest="debug", help="Setting the logging level to DEBUG", type=bool, default=False)
+    args = parser.parse_args()
+    logger = logging.getLogger(__name__)
+    if (args.debug):
+        logging.basicConfig(filename='agents.log', format='%(asctime)s %(levelname)s [%(name)s]: %(message)s', encoding='utf-8', level=logging.DEBUG)
```

Comment:
To be consistent with the other logging that we've introduced yesterday/today (e.g., see https://github.com/wise-agents/wise-agents/pull/403), I think this file should be added to a `log` directory.

Reply:
ok